pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["flit", "setuptools"]
-build-backend = "flit.buildapi"
+requires = ["flit_core >=2,<4", "setuptools"]
```

Comment:
I've left it alone, but you likely don't need both flit_core & setuptools here - they are different options that you wouldn normally pick one of.